reee.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Print("r")
+
+	for true {
+		fmt.Print("e")
+	}
+}
```

Comment:
It's preferable to add a new line here because a file without the new line character at the end is not a text file due to the POSIX standard.